### PR TITLE
PRIME-1273 Send email on admin (site registration)

### PR DESCRIPTION
**Pull request by johnnyl6**
- created send email dialog component for site registration admin

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-container/site-registration-container.component.ts [hunk 1]
```diff
@@ -105,6 +107,24 @@ export class SiteRegistrationContainerComponent implements OnInit {
...
+    const data: DialogOptions = {
+      title: 'Send Email',
+      data: { 'siteId': siteId }
+    };
+
+    this.busy = this.dialog.open(SendEmailComponent, { data })
+      .afterClosed()
+      .pipe(
+        exhaustMap((result: string) => {
+          if (!result) { return EMPTY; }
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-container/site-registration-container.component.ts:120]: Could be reduced down to one line `exhaustMap((result: string) => (result) ? of(result) : EMPTY)`

File: prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <span>Send email to:</span>
+
+  <div class="row mt-4">
+    <div class="col-auto">
+      <button mat-stroked-button
+              cdkFocusInitial
+              (click)="onCancel()">Cancel</button>
+    </div>
+    <div class="col-auto">
+      <mat-card>
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html:19]: The equivalent drop down menu for `Send To` is implemented in the `AdjudicationActionsComponent` where `Adjudicator Actions` is a `mat-flat-button` with a `mat-menu` trigger.

Comment (johnnyl6) [reply prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html]: Changed the "Send To" to a drop down menu button

File: prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  <div class="mt-4">
+    <button mat-stroked-button
+            class="mr-4"
+            cdkFocusInitial
+            (click)="onCancel()">Cancel</button>
+
+    <button mat-flat-button
+            color="primary"
+            class="mb-5 d-inline"
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html:20]: Buttons are inline-block natively and we won't need margin on the bottom.  Can you remove the `class="mb-5 d-inline"` then this PR is good to go

**Review by mtpultz — APPROVED** [review]
image

**Review by AnaisHebert — COMMENTED** [review]
image
Should we remove the Contact Signing Authority on the adjudicator actions now that we have the other email option?

**Pull request merged**